requirements.txt

```diff
@@ -1,3 +1,3 @@
 boto3==1.16.40
 kubernetes==12.0.1
-PyYAML==5.3.1
+PyYAML==5.4
```

Comment:
This update breaks our e2e test container. It now attempts to use Cython to install YAML bindings, which breaks when attempting to use `g++` and `python3-devel` in the `alpine` base.
I have not looked into possible resolutions, but for the moment I will not be merging this change.